### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by CodeWithNotepad** [FlyingDance1995/BrutoForce-Seed#4]
Firstly, thank you for this application but there are two weird things:
1- When I start appication, it shows nearly 69.000.000 checked wallets.
2- Again when I start application, it shows 1 found wallet.
<details><summary>Details About 2. Problem:</summary>
<p>
I looked that wallet. There was a wallet and it had a name. Probably someone had withdrawn that money. But why does the application show that wallet? And does the application really work? Do you have any idea?

</p>
</details> 
When you are writing your comment, please do not hesitate. I am looking forward to your comment.

**Comment by Praveenbabu9392** [comment]
> Firstly, thank you for this application but there are two weird things: 1- When I start appication, it shows nearly 69.000.000 checked wallets. 2- Again when I start application, it shows 1 found wallet.
> 
> Details About 2. Problem:
> When you are writing your comment, please do not hesitate. I am looking forward to your comment.

you have premium key ?

**Comment by CodeWithNotepad** [comment]
No. Why did you ask?

**Comment by Praveenbabu9392** [comment]
> No. Why did you ask?

i think its a scam

**Comment by CodeWithNotepad** [comment]
> i think its a scam
>
No, it isn't scam. And I asked questions. Is the application real? Did you try the application? I tried and it doesn't work. So I asked whether it works. Because of that, does my comment mean that a scam comment?

**Comment by Praveenbabu9392** [comment]
Bud chill out . Its just a reply

**Comment by CodeWithNotepad** [comment]
@Praveenbabu9392 I want to tell something. I think you are trying to find applications that finds crypto wallet, like me. I looked your history. If you find an applicaiton, can you give link?

**Comment by CodeWithNotepad** [comment]
@Praveenbabu9392 Do you have link?. Because I don't want give my Discord.

**Comment by CodeWithNotepad** [comment]
Will you want money?

**Comment by Praveenbabu9392** [comment]
Yeah brother. I purchased it for 200$ why would I give it for free of cost

**Comment by CodeWithNotepad** [comment]
Does it work?

**Comment by Praveenbabu9392** [comment]
It takes time dawg . But i cant promise you anything . The guy who I bought
it from sent me from proofs . If you want i can send them to you

**Comment by CodeWithNotepad** [comment]
Thank you for this conversation but 200$ is too expensive for me. I hope you find some money.

**Comment by Praveenbabu9392** [comment]
I didn’t say i would sell it for 200 dawg . Thats why i asked you to
message me in discord so we can negotiate

**Comment by CodeWithNotepad** [comment]
No, thanks. I don't want buy software. If you find something, please write here. Let me know

**Comment by Praveenbabu9392** [comment]
Alright brother. I was collecting these software ls too plz let me know if
you found any

**Comment by CodeWithNotepad** [comment]
Okay, then I am looking your history.

**Comment by CodeWithNotepad** [comment]
@FlyingDance1995 Is this software real? You didn't say anything.

**Comment by Praveenbabu9392** [comment]
Idk bro . Im trying to install it

**Comment by CodeWithNotepad** [comment]
@Praveenbabu9392 You don't have to answer this question. And are you still trying to install it?

**Comment by CodeWithNotepad** [comment]
@FlyingDance1995 Can you look here?
